admin/en.json

```
@@ -188,6 +188,8 @@
     "TELEMETRY_SUCCESS": "Telemetry Configuration was saved successfully",
     "TELEMETRY_ERROR_STATUS": "Error",
     "TELEMETRY_ERROR_MESSAGE": "Your configuration could not be saved. Please try again.",
+    "DIAGNOSTICS": "Diagnostics",
+    "DIAGNOSTICS_INTRODUCTION": "To aid problem diagnosis by enabling customer facing teams to quickly understand the environment/distribution a user is running.",
```

```suggestion
    "DIAGNOSTICS_INTRODUCTION": "The diagnostics data allows to understand the environment to aid problem diagnosis.",
```
This introduction sounds like a user story from a ticket description but might not make much sense when a user reads it.